docker/zap-api-scan.py

```diff
@@ -103,6 +103,7 @@ def usage():
     print('    -n context_file   context file which will be loaded prior to scanning the target')
     print('    -p progress_file  progress file which specifies issues that are being addressed')
     print('    -s                short output format - dont show PASSes or example URLs')
+    print('    -S                Safe mode; this will skip the active scan and perform a baseline scan
```

Comment:
Nit - 'safe' mode - all of the other descriptions start with a lowercase letter (apart from ZAP which is always in uppercase;)